Assets/ConnectApp/Plugins/QRScanPlugin.cs

```diff
@@ -3,19 +3,20 @@
 using System.Runtime.InteropServices;
 using System.Web;
 using ConnectApp.Api;
+using ConnectApp.Components;
 using ConnectApp.Main;
 using ConnectApp.redux;
 using ConnectApp.redux.actions;
 using ConnectApp.Utils;
 using Unity.UIWidgets.engine;
 using Unity.UIWidgets.external.simplejson;
 using Unity.UIWidgets.foundation;
+using Unity.UIWidgets.material;
```

Comment:
necessary ? we do use stuffs from the material theme?

Reply:
yes, that used the Icons from the material. because we haven't designed.